AberrantSMPP/ASyncSocketClient.cs

```diff
@@ -291,10 +304,16 @@ public void Disconnect()
 					Helper.ShallowExceptions(() => _TcpClient.Close());
 				}
 
+				if (_SslStream != null)
+				{
+					Helper.ShallowExceptions(() => _SslStream.Close());
```

Comment:
Closing the sslStream should be done before closing the TpClient (and hence it's internal stream).

Reply:
Okey, the private field holding the Stream, is closed before closing the TcpClient.